pkg/ansible/controller/status/utils_test.go

```diff
@@ -149,7 +150,8 @@ func TestGetCondition(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			ac := GetCondition(tc.status, tc.condType)
 			if !reflect.DeepEqual(ac, tc.expectedCondition) {
-				t.Fatalf("Condition did no match expected:\nActual: %#v\nExpected: %#v", ac, tc.expectedCondition)
+				t.Fatalf(
+					"Condition did no match expected:\nActual: %#v\nExpected: %#v", ac, tc.expectedCondition)
```

Comment:
I think here we could apply the suggestion as well.

Reply:
ok

Reply:
Solved